messytables/headers.py

```diff
@@ -1,5 +1,8 @@
 from collections import defaultdict
-from itertools import izip_longest
+try:
+    from itertools import zip_longest
+except ImportError:
+    from itertools import izip_longest as zip_longest
```

Comment:
I think we can replace this block with `from compat23 import izip_longest` (then replace `zip_longest`)